### PR TITLE
COMMERCE-6920 Always display normalized discount percentage.

### PR DESCRIPTION
**Pull request by brianikim**
https://issues.liferay.com/browse/COMMERCE-6920

### CODE REVIEW

**Comment by brianikim** [comment]
ci:test:relevant

**Comment by brianikim** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-order-content-web/src/main/resources/META-INF/resources/placed_orders/view_commerce_order_details.jsp [hunk 1]
```diff
@@ -273,12 +273,14 @@ if (commerceOrder != null) {
 
 						<%
 						CommerceMoney shippingDiscountAmountCommerceMoney = shippingCommerceDiscountValue.getDiscountAmount();
+
+						CommerceDiscountValue shippingCommerceDiscountWithTaxValue = commerceOrderPrice.getShippingDiscountValueWithTaxAmount();
```
Comment (dsitu) [inline modules/apps/commerce/commerce-order-content-web/src/main/resources/META-INF/resources/placed_orders/view_commerce_order_details.jsp:277]: We should name this `shippingDiscountValueWithTaxAmount`.
```suggestion
						CommerceDiscountValue shippingDiscountValueWithTaxAmount = commerceOrderPrice.getShippingDiscountValueWithTaxAmount();
```

File: modules/apps/commerce/commerce-order-content-web/src/main/resources/META-INF/resources/placed_orders/view_commerce_order_details.jsp [hunk 2]
```diff
@@ -273,12 +273,14 @@ if (commerceOrder != null) {
...
 						%>
 
 						<dt><liferay-ui:message key="delivery-discount" /></dt>
 						<dd class="text-right"><%= HtmlUtil.escape(shippingDiscountAmountCommerceMoney.format(locale)) %></dd>
 						<dt></dt>
-						<dd class="text-right"><%= HtmlUtil.escape(commerceOrderContentDisplayContext.getLocalizedPercentage(shippingCommerceDiscountValue.getDiscountPercentage(), locale)) %></dd>
+						<dd class="text-right"><%= HtmlUtil.escape(commerceOrderContentDisplayContext.getLocalizedPercentage(shippingCommerceDiscountWithTaxValue.getDiscountPercentage(), locale)) %></dd>
```
Comment (dsitu) [inline modules/apps/commerce/commerce-order-content-web/src/main/resources/META-INF/resources/placed_orders/view_commerce_order_details.jsp:283]:
```suggestion
						<dd class="text-right"><%= HtmlUtil.escape(commerceOrderContentDisplayContext.getLocalizedPercentage(shippingDiscountValueWithTaxAmount.getDiscountPercentage(), locale)) %></dd>
```

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by brianikim** [comment]
hey @dsitu , forgot to update you. The change you requested has been implemented.

**Review by dsitu — APPROVED** [review]

**Comment by igorbeslic** [comment]
ci:help

**Comment by brianikim** [comment]
https://github.com/liferay-commerce/liferay-portal/pull/2483

**Pull request closed without merging**